bundles/org.openhab.binding.souliss/README.md

```diff
@@ -0,0 +1,352 @@
+# Souliss Binding
+
+[Souliss](http://www.souliss.net/) is a networking framework for Arduino and compatibles boards, and is designed to let you easily build a smart home that is distributed over multiple boards via Ethernet, WiFi, wireless point-to-point and RS485 bus. 
+
+Souliss is an open-source and community driven project, you can use the [wiki](https://github.com/souliss/souliss/wiki) and [Community](https://github.com/souliss/souliss/wiki/Community) to get help and share your results.  
+
+## Prerequisites
+
+The binding requires a deployed network.
+As a minimum, you need one Souliss node with Ethernet or WiFi access configured as a [Gateway](https://github.com/souliss/souliss/wiki/Gateway).
+A Gateway is a special node that is able to communicate with the user interfaces.
+The binding interacts as a user interface for Souliss.
+
+A starting point is the [Souliss wiki](https://github.com/souliss/souliss/wiki).
+The best is to start with a single node and connect with SoulissApp.
+The code for networking activities of this binding is based on [SoulissApp](https://github.com/souliss/souliss/wiki/SoulissApp) code, so once connected with SoulissApp, you can move to openHAB directly.
+
+You can use SoulissApp and the Souliss binding at the same time, and generally up to five (by default, but can be increased) user interfaces simultaneously.
+
+### Sketches
+
+The easiest way is start with a simple example to control an ON/OFF light (though a relay). 
+You can go to project [Souliss[(https://github.com/souliss/souliss), see a lot of examples sketches: [Souliss examples](https://github.com/souliss/souliss/tree/friariello/examples)
+
+## Binding Configuration
+
+This binding does not require any special configuration.
+
+## Discovery
+
+This binding can automatically discover devices. 
+First Gateway Node, then Peer Nodes. 
+
+## Supported Things
+
+In Souliss Framework a Typical is one of predefined logic dedicated to smart home devices like lights, heating or antitheft. 
+
+Typical can be one of T11, T12, T13, T14, etc... 
+
+It are defined [here](https://github.com/souliss/souliss/wiki/Typicals).
+
+Typicals match directly with openHAB Thing type.
+
+|Device type |Typical Code | Thing type | 
+|------------|---------|-------------------------------|
+|ON/OFF Digital Output with Timer Option|T11|souliss:t11|
```

Comment:
My comment is about the 'Thing type' column and applies to them all. 
souliss:t11
in this case the thing type is 't11' and when doing textual config the part before the : refers to the binding name. Since it is not part of the 'thing type' name itself I see this as confusing and not how the other bindings present the data.

Reply:
renamed to simple thing type